src/mono/msbuild/apple/build/AppleApp.targets

```diff
@@ -17,6 +23,105 @@
     <PropertyGroup>
       <_MobileIntermediateOutputPath>$([MSBuild]::NormalizeDirectory($(IntermediateOutputPath), 'mobile'))</_MobileIntermediateOutputPath>
     </PropertyGroup>
+    <PropertyGroup>
+      <BundleDir>$(AppleAppBundleDir)</BundleDir>
+      <_MonoHeaderPath>$(MicrosoftNetCoreAppRuntimePackNativeDir)include\mono-2.0</_MonoHeaderPath>
+      <_AotModuleTablePath>$(AppleAppBundleDir)\modules.m</_AotModuleTablePath>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <_TargetsDevice Condition="'$(TargetOS)' == 'ios' or '$(TargetOS)' == 'tvos'">true</_TargetsDevice>
+    </PropertyGroup>
+
+    <!-- 
+      LibraryBuilder takes the approach of accepting items to help what is being linked
+      as opposed to figuring it out in the task.
+    -->
+
+    <!-- common linker arguments for app and library builds -->
+    <ItemGroup>
+      <_CommonLinkerArgs Include="-framework Foundation" />
+      <_CommonLinkerArgs Include="-framework GSS" />
```

Comment:
Probably should be removed until we are ready. The intent was to supply user built framework projects this way.

Reply:
The common linker args part makes it easier to manage changes in the templated cmake file. `%FrameworksToLink%` is something different in that it was intended to supply user defined framework projects like the one to house the one from library mode. It can be removed until we are ready.